build/common.sh

```diff
@@ -557,6 +557,12 @@ function kube::build::ensure_data_container() {
     kube::build::destroy_container "${KUBE_BUILD_DATA_CONTAINER_NAME}"
     ret=1
   fi
+  # This is a hack to force things like jenkins to rebuild their data
+  # containers.
+  if [[ "${ret}" == 0 ]]; then
```

Comment:
Should we version the data container?  Embed that in the name of the container?  Is the assumption that there will be another PR that removes this once jenkins runs?